model/registry/cloudfoundry.yaml

```diff
@@ -0,0 +1,158 @@
+          A guid or another name describing the event source.
+        note: |
+          CloudFoundry defines the `source_id` in the [Loggegator v2 envelope](https://github.com/cloudfoundry/loggregator-api#v2-envelope).
+          It is used for logs and metrics emitted by CloudFoundry. It is
+          supposed to contain the component name, e.g. "gorouter", for
+          CloudFoundry components.
+
+          When system components are instrumented, values from the
+          [Bosh spec](https://bosh.io/docs/jobs/#properties-spec)
+          should be used. The `system.id` should be set to
+          `spec.deployment/spec.name`.
+        examples:
+          - 'cf/gorouter'
+      - id: system.instance.id
+        type: string
+        stability: experimental
+        brief: >
+          A guid describing the concrete instance of the event source.
+        note: |
+          CloudFoundry defines the `instance_id` in the [Loggegator v2 envelope](https://github.com/cloudfoundry/loggregator-api#v2-envelope).
+          It is used for logs and metrics emitted by CloudFoundry. It is
+          supposed to contain the vm id for CloudFoundry components.
+
+          When system components are instrumented, values from the
+          [Bosh spec](https://bosh.io/docs/jobs/#properties-spec)
+          should be used. The `system.instance.id` should be set to `spec.id`.
+        examples:
+          - '218fc5a9-a5f1-4b54-aa05-46717d0ab26d'
+      - id: app.name
+        type: string
+        stability: experimental
+        brief: >
+          The name of the application.
+        note: |
+          Application instrumentation should use the value from environment
+          variable `VCAP_APPLICATION.application_name`. This is the same value
+          as reported by `cf apps`.
+        examples: ['my-app-name']
+      - id: app.id
+        type: string
+        stability: experimental
+        brief: >
+          The guid of the application.
+        note: |
+          Application instrumentation should use the value from environment
+          variable `VCAP_APPLICATION.application_id`. This is the same value as
+          reported by `cf app <app-name> --guid`.
+        examples: ['218fc5a9-a5f1-4b54-aa05-46717d0ab26d']
+      - id: app.instance.id
+        type: string
+        stability: experimental
+        brief: >
+          The index of the application instance. 0 when just one instance is active.
+        note: |
+          CloudFoundry defines the `instance_id` in the [Loggegator v2 envelope](https://github.com/cloudfoundry/loggregator-api#v2-envelope).
+          It is used for logs and metrics emitted by CloudFoundry. It is
+          supposed to contain the application instance index for applications
+          deployed on the runtime.
+
+          Application instrumentation should use the value from environment
+          variable `CF_INSTANCE_INDEX`.
+        examples:
+          - '0'
+          - '1'
+      - id: space.name
```

Comment:
is it similar to common `service.namespace` attribute ?

Reply:
This is really something different. See my explanation in the app name. The OpenTelemetry service attributes really do not fit well with the CloudFoundry trio of org -> space -> app to describe application workloads.